converter/lookml/dimension.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, Optional, Tuple
+
+from dbt_semantic_interfaces.type_enums.time_granularity import TimeGranularity
+from dbt_semantic_interfaces.implementations.elements.dimension import (
+    PydanticDimensionTypeParams,
+)
+from dbt_semantic_interfaces.type_enums.dimension_type import DimensionType
+from dbt_semantic_interfaces.implementations.elements.dimension import (
+    PydanticDimension,
+)
+from pydantic import BaseModel
+
+DEFAULT_TIME_GRANULARITY = TimeGranularity.DAY
+
+
+class LkmlDimension(BaseModel):
```

Comment:
Nit: it's not much longer to call this `LookMLDimension` but might be helpful in avoiding confusion about this class if people don't read the whole `README`!

Reply:
`Lkml` is a common contraction for LookML (the file type is `.lkml`), but I agree, that this could be confusing for someone who is coming to this repo for a reason other than LookML.